### DISCUSSION TITLE
What do color of boxes mean on the "Universe" page?

### DISCUSSION THREAD

**Discussion by etiennebacher** [typst/typst#3762] [General]
Hi, the new "Universe" page is super useful and looks great! However I'm a bit confused about the color of boxes:

![image](https://github.com/typst/typst/assets/52219252/16c3dce8-a9d5-4734-bcdc-e125ee4e9e47)

I don't understand the color code, could you explain it somewhere, maybe under the search bar?

That's very minor, docs are great overall, thanks!

**Comment by laurmaedje** [comment]
All colorful packages are featured by us (those are packages we especially recommend), but there is no difference between the colors.

**Reply by etiennebacher** [reply]
Thanks for the clarification 👍
